src/__tests__/lib/utils/SynapseClient.test.tsx

```diff
@@ -1,208 +0,0 @@
-import { fail } from 'assert'
-import { BackendDestinationEnum } from '../../../lib/utils/functions/getEndpoint'
-import {
-  BatchFileRequest,
-  FileHandleAssociateType,
-  PaginatedResults,
-} from '../../../lib/utils/synapseTypes/'
-import { SynapseClient } from '../../../lib/utils/'
-import { FunctionReturningPaginatedResults } from '../../../lib/utils/SynapseClient'
-
-describe('it works at integration level testing', () => {
```

Comment:
Dropped all of these since they relied on 
* Synapse to be up (at least that's a pretty safe assumption 🙂)
* Synapse to be configured a certain way such that certain users and entities exist and are publicly viewable

I don't think we need a service test.